ui/imports/shared/popups/addaccount/stores/AddAccountStore.qml

```diff
@@ -12,6 +12,8 @@ BasePopupStore {
     required property var addAccountModule
     required property var emojiPopup
 
+    property var mainModuleInstance: mainModule
```

Comment:
better way is to do this 

```
    Component.onCompleted: {
       mainModule.resolvedENS.connect(root.resolvedENS)
    }
```
as per guide https://github.com/status-im/status-desktop/blob/master/CONTRIBUTING.md

Reply:
good point, updated